kivy/tools/packaging/pyinstaller_hooks/pyi_rth_kivy.py

```diff
@@ -5,7 +5,8 @@
 
 os.environ['KIVY_DATA_DIR'] = os.path.join(root, 'data')
 os.environ['KIVY_MODULES_DIR'] = os.path.join(root, 'modules')
-os.environ['GST_PLUGIN_PATH'] = os.path.join(sys._MEIPASS, 'gst-plugins')
+os.environ['GST_PLUGIN_PATH'] = '{}{}{}'.format(
+    sys._MEIPASS, os.pathsep, os.path.join(sys._MEIPASS, 'gst-plugins'))
```

Comment:
Why X/(X/Y) ? (X=_MEIPASS, Y=gst-plugins) ?

Reply:
`os.pathsep` is not the directory join (`/`), this is `X;X/Y`. because both directories are on the path. On windows, the plugins are in the root.

Reply:
I.e. on windows the plugins are in the same directory as the main binaries. We don't separate them out. So pyinstaller puts all the deps binaries in `X`, so it should be in the plugin path.